examples/simple/main.tf

```diff
@@ -18,11 +18,17 @@ provider "google" {
   version = "~> 3.0"
 }
 
+# [START cloudrouter_create]
 module "cloud_router" {
-  source = "../../"
+  source  = "terraform-google-modules/cloud-router/google"
+  version = "~> 0.4"
 
-  name    = "test-router"
+  name   = "test-router"
+  region = "us-central1"
+
+  # project = "my-project-id"
```

Comment:
Would we swap `var.project` with "my-project-id" for docs?

Reply:
No, this is just to provide a sample value.